content/articles/blockchain-as-a-form-of-e-voting/index.md

```diff
@@ -0,0 +1,83 @@
+### Table of contents
+- Introduction
+- Blockchain
+    - Features of Blockchain
+    - Peer-to-Peer network
+    - Transaction
+    - The Chain and the Blocks
+    - Mining and Proof of Work
+- E-voting and Blockchain: Advantages and Disadvantages
+- A Real-Life Example: How Moscow Implemented a Blockchain Vote
+- Conclusion 
```

Comment:
Please note that these should be links. When a reader clicks on it, it should take the reader to the respective section. Check out this [article](https://github.com/section-engineering-education/engineering-education/pull/4148/files) for reference.